### DISCUSSION TITLE
Indefinite pause

### DISCUSSION THREAD

**Discussion by peterantypas** [peterantypas/maiana#146] [General]
As of today, I have to put MAIANA on an indefinite pause. I will no longer take new orders for kits, but I will certainly complete all the orders I have in the queue.

My work/life circumstances have changed and I have to devote about 100% of my attention to something else. Up until today I thought I would be able to keep going, but today I realized that I greatly underestimated the complexity of the project I signed up for. Oh well ... such is life.

**Comment by muebau** [comment]
Very understandable. Good luck and strength for the tasks ahead.

A big thank you for the impressive work and the creation of the first OpenSource AIS!
You have not only created it, improved it over the years, but developed it to the stage of a product.

This is a very rare quality in the community so thank you for your efforts over the years.

**Comment by mickymills** [comment]
Thanks Peter,
Although I never got around to purchasing a kit, I have followed the project with interest for years.



From: Peter Antypas ***@***.***>
Date: Thursday, 10 October 2024 at 12:05 pm
To: peterantypas/maiana ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [peterantypas/maiana] Indefinite pause (Discussion #146)

As of today, I have to put MAIANA on an indefinite pause. I will no longer take new orders for kits, but I will certainly complete all the orders I have in the queue.

My work/life circumstances have changed and I have to devote about 100% of my attention to something else.

**Comment by gah67** [comment]
Thanks for everything you have done Peter - I was one of the very early adopters and it's been great watching the project mature. 

Good luck for the future 

All the best 
Guy

**Comment by hecko** [comment]
Thanks for a great inspiration!

**Comment by Peter-Petrik** [comment]
Thanks from one of your early adopters and testers. The efforts were well worth it, and I strongly believe the results were an incredible accomplishment. We've used MAIANA for thousands of sailed miles.

**Comment by janjaapmeijer** [comment]
Such a shame, that I realized this just too late. What a wonderful project, if you do decide somewhere in the future to continue this endeavour, then please reach out, I'd love to order one!

**Comment by RobinBocquet** [comment]
Hello
Sorry to hear you have to stop. I haven't had time to order a base kit. If you still have a board of the Maïana base kit, without the antenna, I am interested because I have a MacArthur hat board and I think I can make an antenna.
Thanks in any case for the work you've done. I wish you all the best in your life.

**Comment by unkn0wn107** [comment]
This is sad to hear you stop the kit production but totally understand you have personal reasons.

@peterantypas is there a chance you can still provide boards ?

Your project is immensely beneficial, as you open-sourced the hardware and documented everything, thank you !

If you cannot provide boards also, is it possible that I look into board production (batch) ?
Costs seem to be reasonable at around a batch of 10.

Is there a drop-in replacement for the Johanson chip antenna ?

Can you provide some directions for flashing / settings after assembly ?

**Comment by tooluser** [comment]
@peterantypas I hold your project up as one of the most precise and thorough I've seen. It was an inspiration. Thanks for all you've done.

**Comment by dinosound** [comment]
This is a wonderful, incredibly impressive project that I followed for years. Thank you for all your hard work @peterantypas. I wish you all the best and really hope that at some point you are able to unpause the project. 
Be well,
Dinosound

**Comment by peterantypas** [comment]
I need to post this here because I don't have time to respond to individual requests.

I get anywhere from 2 to 5 emails a week from people who are interested in building MAIANA. I'm going to share a couple of facts here that are important, so as to not waste people's time:

1. At its peak, MAIANA demand was about 20 units a month. 
2. There is no possibility of obtaining an FCC license for an Open Source radio. This is explicitly prohibited in the rules. You either close source it or keep it unlicensed.
3. There is little to no profit from this hardware, unless you either raise prices significantly or do all the work in house (including SMT) like I did.

Now, please reconsider your plans.

**Comment by Zaterdag01** [comment]
> As of today, I have to put MAIANA on an indefinite pause. I will no longer take new orders for kits, but I will certainly complete all the orders I have in the queue.
> 
> My work/life circumstances have changed and I have to devote about 100% of my attention to something else. Up until today I thought I would be able to keep going, but today I realized that I greatly underestimated the complexity of the project I signed up for. Oh well ... such is life.

Dear Peter, we might be interested to take over and continue the good work you did for this community, although we need some extra information. Is it possible to have telcon to discuss some matters for realizing your earlier project? Best regards Martin & Hans

**Comment by unkn0wn107** [comment]
Thank you, Peter, for explaining the challenges and economics of Maiana production.

I am also one of those who have tried to reach out for more information about it. :)

We're not looking for a gold mine; we just want to continue producing this interesting hardware and reinvest profits back, if any,  into the Maiana project. Our goal is to have fully working boards to supply interested people who don't have time for the hardware build. Having one myself would also provide a development board for potential firmware contributions.

We're in the early stages. I recently ordered components to produce two test boards (11.9.1) and two antennas. There's still a lot to do before we can extensively test the board, flash the STM32, and test the complete assembly. Also FCC's limitation to closed-source code must have been a very frustrating blocker ... However, we would start with European norms and regulations and do hope there is no such limitation that would change the very open nature of Maiana.

We are open to talking with anyone more or less on this path to seek collaboration opportunities. Where can we reach you, Martin & Hans ?

I have some questions others might also be interested in:
- Are you open to discussing the use of the Maiana name for boards/kits not produced by yourself?
- Is 11.9.1 a "stable" version we should build?
- What are the priorities for hardware or firmware improvements?

**Reply by astuder** [reply]
A heads up for anyone thinking about formally certifying MAIANA: Regulations for AIS transponders also include requirements for the RX side, which the Si446x radio ICs unfortunately can't meet. Specifically adjacent channel selectivity and intermodulation response rejection are insufficient. I think this is due to the narrow-band nature of AIS vs the wider bandwidth that ISM radio ICs are typically designed for.

Plain AIS receivers skirt this issue because receive-only devices are not regulated beyond the usual EMC/EMI requirements.

**Reply by unkn0wn107** [reply]
Thank you for bringing this out, I can see this statement in the Si446x datasheet :
> The 69 dB adjacent channel selectivity with 12.5 kHz channel spacing ensures robust receive operation in harsh RF conditions, which is particularly important for narrowband operation

You mean that the spacing is too low ? Or several specs are insufficient to meet marine radio standard needs ?

Marine radio requirements feel really steep just looking at the number of intricated documents and their page numbers ...

Yes, I also don't exclude the same strategy Peter used in the past of providing as receiver with an empty radio slot on the pcb to plug at own discretion.

**Reply by astuder** [reply]
I can't share the full document, but I will leave this here:
![rx-requirements](https://github.com/user-attachments/assets/0c1f31cf-900a-4d63-9123-85f0c22e5cc0)

**Reply by unkn0wn107** [reply]
Thank you very much for this really interesting testing results ! I sent you an invitation on LinkedIn to eventually catch-up :)

**Comment by unkn0wn107** [comment]
I just created a new thread specifically to discuss about ideas / projects to [bring Maiana back to production](https://github.com/peterantypas/maiana/discussions/148).
